tests/Integration/Database/EloquentDeleteTest.php

```diff
@@ -59,7 +45,11 @@ public function testOnlyDeleteWhatGiven()
         Post::latest('id')->limit(1)->delete();
         $this->assertCount(9, Post::all());
 
-        Post::join('comments', 'comments.post_id', '=', 'posts.id')->where('posts.id', '>', 1)->orderBy('posts.id')->limit(1)->delete();
+        Post::join('comments', 'comments.post_id', '=', 'posts.id')
+            ->where('posts.id', '>', 8)
```

Comment:
SQLite and MySQL behave different here so we should adjust the test accordingly.